deploy/charts/crate-operator/values.yaml

```diff
@@ -0,0 +1,68 @@
+---
+# Default values for crate-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# If the `crate-operator-crds` is already installed, its installation
+# can be disabled by changing this value to `false`.
+crate-operator-crds:
+  # Make this disabled if you don't want to install
+  # the CRD dependency automatically.
+  enabled: true
+
+replicaCount: 1
+
+image:
+  repository: crate/crate-operator
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
```

Comment:
In the deployment script (jenkins) it is overwritten ( `--set image.tag=${APP_VERSION}` )
I don't think it is a problem to let the default value as `latest` otherwise we will have to maintain it here as well.

Reply:
btw, maybe worth having this commented out then? We already need to maintain the chart `AppVersion`, and if the tag is not specified - it will default to that :thinking: